src/utils/cookie.ts

```diff
@@ -35,7 +36,16 @@ export const parse = (cookie: string, name?: string): Cookie => {
   const parsedCookie: Cookie = {}
   const unsingedCookies = _parseCookiePairs(cookie, name).filter((pair) => {
```

Comment:
It's typo! `unsingedCookies` should be `unsignedCookies`.

Reply:
Thanks, fixed.